starter_code/css/desktop.css

```diff
@@ -0,0 +1,15 @@
+img{
+  max-width: 100%;
+    height: auto;
+}
+
+
+@media (min-width: 1280px){
+	section{ 
+		font-size: 36px; 
+	}
+	.post{
+    font-size:36px;
+}
+
```

Comment:
You're missing a close `}` here to close out the `@media (min-width: 1280px){`

It should look like this:

```css
@media (min-width: 1280px){
   section {
      font-size: 36px;
   }

   .post {
      font-size: 36px;
   }
}
```

Also note the indentation in mine relative to yours. The way I've written is much easier to read as you can clearly match up the opening `{` and closing `}` brackets